### ISSUE TITLE
Fix Publishing Dates

### ISSUE THREAD

**Issue by samuei** [hackerkid/Mind-Expanding-Books#208]
I don't know offhand when Jane Austen published Sense and Sensibility, but I *strongly* suspect it was not 2003. Leo Tolstoy, having died in 1910, did not release much in the 2010s.

A _lot_ of these dates are wildly incorrect. I suspect this may have been done automatically, perhaps grabbing whatever date GoodReads has, but clearly the information gleaned was incorrect and needs to be reviewed more thoroughly.

**Comment by hackerkid** [comment]
@samuei You are right. The year of publicatoon of majority of the book in the list was extracted automatically using the API. Someone has to go through them manually and fix the incorrect ones.

**Comment by kyebrowning** [comment]
Are you referring to the dates on the readme/readme-new? Would you mind if I update them to their correct publish date?

**Comment by Firsh** [comment]
Future Crimes by Marc Goodman was not published in 1990
